tests/test_edge_cases.py

```diff
@@ -144,24 +164,26 @@ async def test_stage_checksum_missmatch(
     # place the content for an example file in the permanent storage:
     file_object = file_fixture.copy(
         update={
-            "bucket_id": joint_fixture.config.staging_bucket,
-            "object_id": EXAMPLE_FILE.file_id,
+            "bucket_id": joint_fixture.staging_bucket,
```

Comment:
This should probably also be outbox instead, might need to doublecheck, though

Reply:
You're right, good catch: [b0dbdf7](https://github.com/ghga-de/internal-file-registry-service/pull/39/commits/b0dbdf776d534d3ae81d2cd8a5c090afb812690b)